test/fixtures/cache/cache.js

```diff
@@ -0,0 +1,16 @@
+const tasksList = require("../tasks/tasks");
+
+// Returns a dummy sample value to be stored in cache
+const getDummyResponse = (expiryTime = 2) => {
+  const apiResponse = { message: "Tasks returned successfully!", tasks: tasksList() };
+  const stringifiedResponse = JSON.stringify(apiResponse);
+
+  return {
+    expiry: new Date().getTime() + expiryTime * 60000,
```

Comment:
I think it's better to declare numbers as constant and use them

Reply:
I was thinking of doing this, but what if we want to test for different expiry times like 2mins, 10mins ...etc?

Reply:
IMO,
DEFAULT_EXPIRY_TIME_MIN
inspite of that
EXPIRY_TIME_IN_MIN is more meaningful.